spec/introduction.md

```diff
@@ -109,18 +119,21 @@ class Test
     }
 }
 ```
-If the program is stored in the file `test.cs`, when `test.cs` is compiled, the `acme.dll` assembly can be referenced using the compiler's `/r` option:
+
+If the program is stored in the file *test.cs*, when *test.cs* is compiled, the *acme.dll* assembly can be referenced using the compiler's `-r` option:
 
 ```console
-csc /r:acme.dll test.cs
+csc -r:acme.dll test.cs
```

Comment:
Since this is a demo, perhaps use the full form of the flag for clarity: `-reference:acme.dll`
(Having `-r` is pretty cryptic.)